agent/util-scripts/pbench-register-tool-trigger

```diff
@@ -69,13 +77,18 @@ while true; do
 	esac
 done
 
-if [ "$start_trigger" != "${start_trigger%%:*}" ]; then
+if [[ -z "$start_trigger" && -z "$stop_trigger" ]] ;then
+	printf "At least one of --start-trigger or --stop-trigger must be specified.\n"
+	usage
+	exit 3
+fi
+if [[ "$start_trigger" != "${start_trigger%%:*}" ]]; then
 	printf "$script_name: the start trigger cannot have a colon in it: \"${start_trigger}\"\n"
-	exit 1
+	exit 4
 fi
 if [ "$stop_trigger" != "${stop_trigger%%:*}" ]; then
 	printf "$script_name: the stop trigger cannot have a colon in it: \"${stop_trigger}\"\n"
-	exit 1
+	exit 5
 fi
```

Comment:
Is it a problem if the start and stop triggers are the same text?  (I think not...but, if I'm wrong, then we should check for that.)

Reply:
Shouldn't be.  That is really at the caller's discretion for the behavior they want.

Reply:
I meant that the Tool Meister code will do the right thing if the two are the same -- i.e., if it encounters the trigger, it will either perform a start (if the tool is not running) or a stop (if the tool is already running), and it won't do something silly like start it a second time or fail to stop it if appropriate....